target/riscv/cpu_bits.h

```diff
@@ -363,6 +363,11 @@
 #define CSR_UCCSR           0x8C0
 #define CSR_SCCSR           0x9C0
 #define CSR_MCCSR           0xBC0
+
+#define CCSR_ENABLE         0x1
+#define CCSR_DIRTY          0x2
```

Comment:
These need some kind of prefix I think, perhaps X given they exist for all the modes.